src/Config.php

```diff
@@ -69,4 +80,23 @@ public function clear(string $cacheDir): void
     {
         $this->cache->clear();
     }
+
+    /**
+     * @return array<mixed>
+     */
+    private function getConfig(): array
+    {
+        if (is_array($this->config)) {
+            return $this->config;
+        }
+
+        $this->config = $this->eventEngine->compileCacheableConfig();
+
+        return $this->config;
+    }
+
+    private function isDevEnv(): bool
+    {
+        return preg_match('/(dev(.*)|local)/i', $this->environment) === 1;
```

Comment:
Why do you use (.*) in the "dev" part of the regex?

Reply:
Also allow `development`, `develop`